recipes/access_private/all/test_package/conanfile.py

```diff
@@ -13,6 +14,6 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self.settings):
+        if not tools.build.cross_building(self, self.settings):
```

Comment:
```suggestion
        if not tools.build.cross_building(self):
```
This is still incorrect

Reply:
Okey, weired. Anyway, it is fixed in a later script version :)